sycl/include/sycl/handler.hpp

```diff
@@ -1244,15 +1244,15 @@ class __SYCL_EXPORT handler {
                   "sycl::parallel_for with sycl::range");
 
 #ifdef SYCL2020_CONFORMANT_APIS
+    static_assert(std::is_convertible_v<item<Dims>, LambdaArgType> ||
+                      std::is_convertible_v<item<Dims, false>, LambdaArgType>,
+                  "sycl::parallel_for(sycl::range) kernel must have an "
+                  "argument of sycl::item type, or of a type which is "
+                  "implcitly convertible from sycl::item");
```

Comment:
```suggestion
                  "implicitly convertible from sycl::item");
```

Reply:
Fixed in f3edff180367d853aa2c2b8cad90aa1e5bf3690b